src/Lesson_6/extraTaskOne/Computer.java

```diff
@@ -0,0 +1,33 @@
+package Lesson_6.extraTaskOne;
+
+public class Computer {
+    private double price;
+    private String model;
+    private RAM RAM;
+    private HDD HDD;
+
+
+    public Computer() {
+        RAM = new RAM("ram", 256);
+        HDD = new HDD("hdd", 156, "внешний");
+    }
+
+    public Computer(double price, String model) {
+        this.price = price;
+        this.model = model;
+    }
+
+    public Computer(double price, String model, RAM RAM, HDD HDD) {
+        this(price, model);
+        this.RAM = RAM;
+        this.HDD = HDD;
+    }
+
+    public void printInfo1() {
+        System.out.println("Computer model: " + model + "\n " + "Price: " + price + "$" + "\n");
+    }
+
+    public void printInfo2() {
+        System.out.println("Computer model: " + model + " " + "Price: " + price + "$" + "\n" + RAM.printInfoRAM() + "\n" + HDD.printInfoHDD());
```

Comment:
we don't need two methods. one of them, that check if ram and hdd are null, is enough